lib/beaker/hypervisor/docker.rb

```diff
@@ -4,6 +4,7 @@ class Docker < Beaker::Hypervisor
     def initialize(hosts, options)
       require 'docker'
       @options = options
+      @options[:set_env] = false
```

Comment:
I'm open to suggestions, but set_env should default to false in docker environments. You can't have a default of

* run sshd as main process, and
* restart sshd

because it just doesn't work, and it's an issue for new users.
I see it more as sent_env defaults to false for Docker and you can override it if you want, because you need to use a custom image or command too